src/main/java/org/json/JSONObject.java

```diff
@@ -272,6 +279,17 @@ public JSONObject(JSONTokener x, JSONParserConfiguration jsonParserConfiguration
         }
     }
 
+    private Object getValue(JSONTokener x, boolean strictMode) {
```

Comment:
These methods won't be needed if you override x.nextValue()  and pass in a JSONParserConfiguration object